src/themes/default/components/core/ProductTile.vue

```diff
@@ -70,24 +85,34 @@ export default {
     }
   },
   computed: {
-    thumbnailObj () {
+    thumbnailObj() {
       return {
         src: this.thumbnail,
         loading: this.thumbnail
       }
+    },
+    favoriteIcon() {
+      return this.isOnWishlist ? 'favorite' : 'favorite_border'
     }
   },
   methods: {
-    onProductPriceUpdate (product) {
+    onProductPriceUpdate(product) {
       if (product.sku === this.product.sku) {
         Object.assign(this.product, product)
       }
     },
-    visibilityChanged (isVisible, entry) {
+    visibilityChanged(isVisible, entry) {
       if (isVisible) {
```

Comment:
While being here, could you refactor this area a little bit to avoid so many nesting levels?